src/repl-frontend/scala/tools/nsc/interpreter/shell/ILoop.scala

```diff
@@ -432,15 +452,12 @@ class ILoop(defaultIn: BufferedReader = null, protected val out: JPrintWriter =
   }
 
   /** interpret all lines from a specified file */
-  def interpretAllFrom(file: File, verbose: Boolean = false) {
-    savingReader {
-      savingReplayStack {
-        file applyReader { reader =>
-          in = if (verbose) new SimpleReader(reader, out, interactive = true) with EchoReader
-               else SimpleReader(reader, out, interactive = false)
-          echo(s"Loading $file...")
-          loop()
-        }
+  def interpretAllFrom(file: File, verbose: Boolean = false): Unit = {
+    savingReplayStack {
+      file applyReader { reader =>
```

Comment:
- [ ] regression -- this did more than changing verbosity, also need to pass in new reader